Replace summary blocker with document tags in document list

maxjmohr opened the pull request:
### Description

Display tags in the workspace overview

## Testing Instructions

Test if the tags are displayed correctly and if everything looks nice

### Type of Change

Please delete options that are not relevant.

- [ ] 🐛 Bug fix
- [x] ✨ New feature
- [ ] 🧹 Code cleanup or refactor

### Screenshots (if applicable)

<img width="3628" height="1295" alt="image" src="https://github.com/user-attachments/assets/fe88a4a4-9164-4e94-ad11-935179c73bb2" />

### Related Issue

closes #231

jzakotnik commented:
Sorry for the delay we should have merged this already.. 
Two little things
- we need spanish translation?
- What do you think about outlined style of the tags? The filled colors are very prominent and aggressive given that it's not the most important information..

maxjmohr commented:
@jzakotnik so I did a rebase on the current `main` branch and changed this:
- Deleted "summaryOf" from spanish messages
- Turned tags background color into a border

Let me know if this is what you imagined :) I also think it isnt as present as before
